__tests__/cli.test.ts

```diff
@@ -0,0 +1,84 @@
+import execa from 'execa';
+import { Project } from 'fixturify-project';
```

Comment:
🤯 I didn't know something like this existed! This is awesome. I've done this by hand more times than I can count...

Reply:
We use it extensively, in countless projects. It's very powerful and allows you to do ad-hoc testing really easily.